capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -0,0 +1,95 @@
+package com.step902020.capstone;
+
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
+import org.springframework.data.annotation.Id;
+import java.time.LocalDateTime;
+import java.util.*;
+
+@Entity(name = "event")
+public class Event{
+
+  @Id
+  Long datastoreID;
+
+  @Field(name="organizationName")
+  String organizationName;
```

Comment:
You can use an `Organization` reference here with `@Reference`.
See: https://cloud.spring.io/spring-cloud-gcp/reference/html/#key-reference-relationships